VideoLocker/res/layout/fragment_course_dashboard.xml

```diff
@@ -1,62 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                xmlns:tools="http://schemas.android.com/tools"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:background="@color/white"
-                android:orientation="vertical" >
-
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@color/white"
+    android:orientation="vertical">
 
-    <com.android.volley.toolbox.NetworkImageView
-        android:id="@+id/header_image_view"
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:minHeight="144dp"
-        android:layout_alignParentTop="true"
-        android:layout_centerHorizontal="true"
-        android:scaleType="fitXY"
-        android:background="@drawable/edx_map_login" />
+        android:layout_height="@dimen/course_card_height">
 
-    <LinearLayout
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:minHeight="80dp"
-        android:orientation="vertical"
-        android:padding="12dp"
-        android:background="@color/white">
+        <ImageView
+            android:id="@+id/header_image_view"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:layout_alignParentTop="true"
+            android:layout_centerHorizontal="true"
+            android:background="@drawable/edx_map_login" />
 
-        <org.edx.mobile.view.custom.ETextView
-            android:id="@+id/course_detail_name"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            style="@style/bold_text"
-            android:textSize="20sp"
-            android:paddingBottom="6dp"
-            tools:text="test course" />
+        <LinearLayout
```

Comment:
Yes, the FrameLayout measuring and layout logic is extremely simple compared to RelativeLayout.
> On 10-Sep-2015, at 6:03 pm, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/res/layout/fragment_course_dashboard.xml <https://github.com/edx/edx-app-android/pull/354#discussion_r39156649>:
> 
> >  
> > -        <org.edx.mobile.view.custom.ETextView
> > -            android:id="@+id/course_detail_name"
> > -            android:layout_width="wrap_content"
> > -            android:layout_height="wrap_content"
> > -            style="@style/bold_text"
> > -            android:textSize="20sp"
> > -            android:paddingBottom="6dp"
> > -            tools:text="test course" />
> > +        <LinearLayout
> Will using FrameLayout instead of RelativeLayout give us some performance improvements ?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/354/files#r39156649>.
>